cypress.config.ts

```diff
@@ -0,0 +1,45 @@
+import { defineConfig } from 'cypress';
+require('dotenv').config();
+
+/**
+ * Filter test spec paths based on env var configuration
+ * @returns
+ */
+const getSpecPattern = (): string[] => {
+  const optionalPaths = [
+    {
+      path:   'cypress/e2e/tests/setup/**/*.spec.ts',
+      active: process.env.TEST_SKIP_SETUP !== 'true'
+    }
+  ];
+  const activePaths = optionalPaths.filter(({ active }) => Boolean(active)).map(({ path }) => path);
```

Comment:
This could be just reduce instead of 2 array operations.

Reply:
Readability is preferred over smaller loops.